README.md

```diff
@@ -44,6 +44,14 @@ Spyder may also be used as a PyQt4/PySide extension library (module
 `spyderlib`). For example, the Python interactive shell widget used in
 Spyder may be embedded in your own PyQt4/PySide application.
 
+## Running from source
```

Comment:
I don't know, maybe using the Readme to help Github users is the best thing to do in 2015 :-)

@Nodd, what do you think?